### PR TITLE
60 61 website

### LINKED ISSUE #61: Add information about the the creators to the site
On the botomir.com website we should provide some information about who we are, why we made botomir what its purpose is, etc.

### PR DESCRIPTION
**Pull request by SoorajModi**
Pulling because i have midterms next week that i want to focus on.

This PR covers:
- Closes #61 
- Adds UI for #60 
- UI improvements to site
- Add about page
- Update temporary Botomir logo
- Fix CSS issues with navbar and footer
- Removes Documentation and Statistics tabs in navbar because they are empty

To-do (in future issues):
- Documentation page
- Query data for stats page in statisticsController.js
- UI needs to be improved for smaller screens (ie. mobile)

### CODE REVIEW

**Comment by MarshallAsch** [comment]
what are the UI improvements that are present in this PR?

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -7,39 +7,56 @@
...
-{{{body}}}
+    <footer>
+        <a href="https://github.com/SoorajModi/Botomir" target="_blank" class="footer-icon">
+            <i class="fa fa-github fa-lg"></i>
+        </a>
+    </footer>
+</div>
 </body>
-<footer>
-    <p>GPL 2.0 License</p>
```
Comment (MarshallAsch) [inline views/layouts/main.handlebars:43]: Should the license / copy-write statement be kept in the footer? 
(Not a change request, but a question)

File: views/about.handlebars [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <div class="container center-content about-container-padding">
+        <!--        <img src="/images/logo.jpg" alt="Botomir Logo">--> <!-- Use Botomir logo when ready -->
+        <h1 class="about-header-padding">About Botomir</h1>
+        <p class="align-text-left about-description-padding">
+            Botomir is a Discord Bot written in Javascript. They were born on November 25, 2020, and was a happy
+            accident more than a planned project. Botomir was created because he is a remedy to loneliness. It can be
+            just you and Botomir, they won't tell.
+
+            Botomir loves to eat chicken and waffles, washing it down with a Four loco, and cookies for dessert. Their
+            favourite song is the windows startup sound. His hobbies include going on long scrolls through Wikipedia,
```
Comment (MarshallAsch) [inline views/about.handlebars:11]: Gendering of botomir, everywhere else gender neutral phrases are used, should this `his` be changed to `their`?

**Review by MarshallAsch — CHANGES_REQUESTED** [review]
All good pending responses to the comments. And moving forwards the 2 issues should be split up into separate PR's and since a lot of stuff for #60 was added here but is disabled.

**Comment by SoorajModi** [comment]
UI improvements include
- Using GitHub font awesome icon, and place it on footer instead of navbar
- Fix issue with footer not staying at bottom of page
- Remove pages without content in navbar
- Tweak CSS in about page so it looks nicer

File: views/layouts/main.handlebars (hunk 1 above)
Comment (SoorajModi) [reply views/layouts/main.handlebars:43]: Its not something we really need.

We should have a user agreement and privacy policy however

File: views/about.handlebars (hunk 2 above)
Comment (SoorajModi) [reply views/about.handlebars:11]: Yes, will change

Comment (SoorajModi) [reply views/about.handlebars]: Fixed

File: views/layouts/main.handlebars (hunk 1 above)
Comment (MarshallAsch) [reply views/layouts/main.handlebars:43]: This has been marked in #69

**Review by MarshallAsch — APPROVED** [review]

**Pull request merged**